### DISCUSSION TITLE
Can login to my Github but im not authorized to make any push commits

### DISCUSSION THREAD

**Discussion by gojrs** [community/community#53471] [Repositories] [answered]
### Select Topic Area

Question

### Body

I am new to this and am trying to make a commit in a new repo. I get fatal: 'github.com/gojrs/go-ui' does not appear to be a git repository in the CLI in the desktop it shows the repo, but I get an similar error. It suggests I logout/in I have.

**Comment by airtower-luna** [comment]
Can you post the *exact and full* Git command you used and its output? It's hard to guess what's going on without that.

Also the output of `git remote -v` might be helpful.

**Reply by gojrs** [reply]
% git push origin main
fatal: 'github.com/gojrs/go-ui' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
% git push --set-upstream gojrs/go-ui main
fatal: 'gojrs/go-ui' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
%

**Reply by airtower-luna** [reply]
That really looks like *something* is wrong with the remote repository URL. What does `git remote -v` say?

**Comment by gojrs** [comment]
origin  github.com/gojrs/go-ui (fetch)
origin  github.com/gojrs/go-ui (push)

**Comment by airtower-luna** [comment, marked as answer]
Yep, those URLs are broken. Most importantly they lack the `https:` scheme, which means they'll be interpreted as directory names (like for another local repository), not HTTPS URLs. You can fix them with [`git remote set-url`](https://git-scm.com/docs/git-remote#Documentation/git-remote.txt-emset-urlem):

```
git remote set-url origin https://github.com/gojrs/go-ui.git
git remote set-url --push origin https://github.com/gojrs/go-ui.git
```

**Reply by gojrs** [reply]
thank you so much. I figured it had to do with my path

**Reply by airtower-luna** [reply]
Glad I could help! Please don't forget to mark the answer. :slightly_smiling_face:

**Comment by gojrs** [comment]
Very help thank you add.
